CrewChiefV4/RF2/RF2GameStateMapper.cs

```diff
@@ -2076,7 +2210,8 @@ private FrozenOrderData GetFrozenOrderData(FrozenOrderData prevFrozenOrderData,
                         var v = scoring.mVehicles[i];
                         if (v.mID == vehToFollowId)
                         {
-                            fod.DriverToFollowRaw = RF2GameStateMapper.GetStringFromBytes(v.mDriverName).ToLower();
+                            fod.DriverToFollowRaw = RF2GameStateMapper.GetStringFromBytes(v.mDriverName).ToLowerInvariant();
```

Comment:
Do we need both?

Reply:
You mean, do we need ToLower and Invariant?  Not really, just a habit ;)

Reply:
actually what i meant was that you are setting fod.DriverToFollow twice here :)